Improve various docstrings

misi9170 opened the pull request:
This PR makes a handful of updates to docstrings throughout FLORIS to respond to minor issues that have been raised.
- Adds details on the serial refine yaw optimizer #1021 
- Adds a simple docstring for `FlorisModel.get_defaults()` #1161
- Updates references to `floris.simulation` (now `floris.core`) #1105 . In this case, I opted to remove a description of the floris input file structure and referred readers to the docs and the default_inputs.yaml for an example.
- Update variable name from `r` to `r_squared` in Gaussian models (see #1102).
- Add references to [Katic et al. (1987)](https://backend.orbit.dtu.dk/ws/portalfiles/portal/106427419/A_Simple_Model_for_Cluster_Efficiency_EWEC_86_.pdf) for sum of squares combination model.
- Updates the description docstring in examples/examples_turbine/001_reference_turbines.py, which I forgot to update in #1146 .
- Fixes hyperlink formatting in READEME

misi9170 committed:
Update references to floris.simulation and add docstring to get_defaults

misi9170 committed:
Improve docstrings for user-level functionality

misi9170 committed:
Update args docstring

misi9170 committed:
Formatting

misi9170 committed:
Update docstring

rafmudaf commented:
Hey @misi9170 I have a few things to add along these lines, if you don't mind waiting a few days for me to submit a PR to your branch

misi9170 commented:
@rafmudaf please do, no rush. Fine also for you to simply commit and push into my branch if you'd like.

misi9170 committed:
Fix WETO software hyperlink

misi9170 commented:
Converting to draft while I'm still planning to address a couple of things:
- [x] Secondary items mentioned in #1105 
- [x] Details on the value of `initial` in the Crespo-Hernández model #773

misi9170 committed:
Fix documentation for cut planes and add documentation for visualizations

misi9170 committed:
Add Katic reference

misi9170 committed:
Switch from r to r_squared to be explicit and avoid confusion

misi9170 committed:
Another reference to Katic

misi9170 committed:
Add notes regarding CH initial exponent

misi9170 committed:
Spelling fix

misi9170 commented:
Given the imminent change of the github organization #1177 , I intend to merge this PR soon and include it in an upcoming patch release for FLORIS v4.6. @rafmudaf and I spoke and decided to hold off on including his docs changes in this PR. Ready for review.

misi9170 commented:
@paulf81 , I've added you as a reviewer; hopefully, that's ok. Can you take a closer look at the documentation I've added for Serial Refine and then a quick pass over the rest?

misi9170 committed:
Merge develop

misi9170 committed:
Square image, update email addresses

paulf81 commented on floris/core/wake_deflection/gauss.py:
```diff
@@ -297,29 +297,29 @@ def wake_added_yaw(
     # NOTE: this is the top of the grid, not the top of the rotor
     zT = z_i - (HH + D / 2) + NUM_EPS  # distance from the top of the grid
-    rT = ne.evaluate("yLocs ** 2 + zT ** 2")  # TODO: This is (-) in the paper
+    rT_squared = ne.evaluate("yLocs ** 2 + zT ** 2")  # TODO: This is (-) in the paper
```
is the TODO still todo?

misi9170 replied:
Hmm, good point. This TODO was already there, but this is as good a time as any to look into it. I'll take a look.

misi9170 replied:
Ok, best I can tell is that the implementation is correct and there is a typo in the paper. I've updated the note to comment on this and removed the `TODO`.

paulf81 replied:
Then sounds good to merge!

paulf81 reviewed (approved):
All changes look good to me, thanks! @misi9170

misi9170 committed:
Update todo note
